agent-control/tests/on_host/scenarios/invalid_remote_config.rs

```diff
@@ -234,6 +232,10 @@ fn onhost_opamp_sub_agent_invalid_remote_config_rollback_previous_remote() {
     let valid_remote_config = "fake_variable: valid from remote\n";
     opamp_server.set_config_response(sub_agent_instance_id.clone(), valid_remote_config);
 
+    #[cfg(target_family = "windows")]
+    // Give it time for AC to figure out the effective config
+    std::thread::sleep(Duration::from_secs(1));
```

Comment:
Does it take longer on Windows? Shouldn't the `retry` below be enough for waiting? 🤔